src/test/java/org/broadinstitute/hellbender/tools/walkers/GnarlyGenotyperIntegrationTest.java

```diff
@@ -28,8 +28,23 @@
  * Created by gauthier on 3/9/18.
  */
 public class GnarlyGenotyperIntegrationTest extends CommandLineProgramTest {
+
+    // If true, update the expected outputs in tests that assert an exact match vs. prior output,
+    // instead of actually running the tests. Can be used with "./gradlew test -Dtest.single=GnarlyGenotyperIntegrationTest"
+    // to update all of the exact-match tests at once. After you do this, you should look at the
+    // diffs in the new expected outputs in git to confirm that they are consistent with expectations.
+    public static final boolean UPDATE_EXACT_MATCH_EXPECTED_OUTPUTS = false;
+
     private static final List<String> NO_EXTRA_ARGS = Collections.emptyList();
 
+    /*
+     * Make sure that someone didn't leave the UPDATE_EXACT_MATCH_EXPECTED_OUTPUTS toggle turned on
+     */
+    @Test
+    public void assertThatExpectedOutputUpdateToggleIsDisabled() {
+        Assert.assertFalse(UPDATE_EXACT_MATCH_EXPECTED_OUTPUTS, "The toggle to update expected outputs should not be left enabled");
+    }
+
```

Comment:
I noted it over in #7122 post merge, but there are a few tests below that were commented out. Not sure if there is a plan to restore these, but I did not update the corresponding ExcessHet values.

Reply:
The tests pass on master if I add them back in. I think I just forgot to put them back.  (I don't have an efficient and elegant method for running the debugger on the Nth test of a data provider if N > 0 so I just commented out 0 through N-1.)  You have my permission to uncomment and update the exact match again.

Reply:
Done.